src/test/scala/sigmastate/eval/BasicOpsTests.scala

```diff
@@ -60,16 +60,11 @@ class BasicOpsTests extends FunSuite with ContractsTestkit with Matchers {
     val c2 = collection[Byte](3, 4)
     c1.append(c2).toArray shouldBe Array[Byte](1, 2, 3, 4)
   }
-  test("examples from wpaper")  {
-    val selfId = collection[Byte](0, 1)
-    val self = new TestBox(selfId, 10, noBytes, noBytes, noBytes, noRegisters)
-    val ctx = testContext(noInputs, noOutputs, height = 200, self, emptyAvlTree, dummyPubkey, Array())
-  }
 
-  test("box.creationInfo._1 is Int") {
-    val box = newAliceBox(1, 100, Map(3 -> toAnyValue((20 -> SigmaDsl.Colls.fromArray(Array.emptyByteArray)))))
-    box.creationInfo._1 shouldBe a [Integer]
-  }
+//  test("box.creationInfo._1 is Int") {
```

Comment:
use ignore, not commenting out the code. Please provide an explanation why the test is ignored currently

Reply:
made it working in upcoming PR